.github/renovate.json

```diff
@@ -1,4 +1,5 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>technologiestiftung/renovate-config#v1.1.1-next.2"]
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>technologiestiftung/renovate-config#v1.1.1-next.2"],
```

Comment:
Does not need to point to a tag can be `github>technologiestiftung/renovate-config`